test/spec/components/Feel.spec.js

```diff
@@ -25,12 +25,9 @@ import {
   DescriptionContext,
```

Comment:
:spiral_notepad: Don't look into the tests, the refactor completely breaks them.